Python/frame.c

```diff
@@ -94,6 +94,17 @@ take_ownership(PyFrameObject *f, _PyInterpreterFrame *frame)
     }
 }
 
+void
+_PyFrame_ClearLocals(_PyInterpreterFrame *frame)
+{
+    assert(frame->stacktop >= 0);
+    for (int i = 0; i < frame->stacktop; i++) {
```

Comment:
I think `frame->stacktop = 0;` needs to come before the loop to prevent the GC seeing freed objects.

Reply:
Let me make a fix for that.